plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1606,15 +1587,23 @@ private void appendTableEntries(
                         createdTime));
     }
 
-    private static void appendAddFileEntries(TransactionLogWriter transactionLogWriter, List<DataFileInfo> dataFileInfos, List<String> partitionColumnNames, boolean dataChange)
+    private static void appendAddFileEntries(TransactionLogWriter transactionLogWriter, List<DataFileInfo> dataFileInfos, List<String> partitionColumnNames, List<String> originalColumnNames, boolean dataChange)
             throws JsonProcessingException
     {
+        Map<String, String> toOriginalColumnNames = originalColumnNames.stream()
+                .collect(toImmutableMap(name -> name.toLowerCase(ENGLISH), identity()));
         for (DataFileInfo info : dataFileInfos) {
             // using Hashmap because partition values can be null
             Map<String, String> partitionValues = new HashMap<>();
             for (int i = 0; i < partitionColumnNames.size(); i++) {
                 partitionValues.put(partitionColumnNames.get(i), info.getPartitionValues().get(i));
             }
+
+            Optional<Map<String, Object>> minStats = toOriginalColumnNames(info.getStatistics().getMinValues(), toOriginalColumnNames);
```

Comment:
`toOriginalColumnNames` has this comment: "Lowercase column names because statistics generated by Trino has lowercase names"
yet, we're operating on `DataFileInfo` that Delta connector created (not engine)

are they lowercase because `ColumnChunkMetaData.getPath` ends up being lowercase in 
https://github.com/trinodb/trino/blob/e26bf49d57cc596014b051485c7e0898d20798d5/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeWriter.java#L218?

please update the code comment in `toOriginalColumnNames`

Reply:
Updated the code comment. Parquet file contains the original names, but it's lowercased when reading the metadata at
https://github.com/trinodb/trino/blob/023f8b4cb5fc83615aae8cb30da31ffd61004603/lib/trino-parquet/src/main/java/io/trino/parquet/reader/MetadataReader.java#L139-L141